src/components/tasks/card/index.tsx

```diff
@@ -48,11 +49,14 @@ const Card: FC<Props> = ({
     const [showEditButton, setShowEditButton] = useState(false);
     const [keyLongPressed] = useKeyLongPressed();
     const [isLoading, setIsLoading] = useState<boolean>(false);
-
     const { actions, state } = useAppContext();
+    const router = useRouter();
+    const { query } = router;
+    const dev = !!query.dev;
```

Comment:
What does the variable `dev` do? If this is related to devMode, how about instead of using the terminology of dev vs prod, use `isNewCardEnabled` kind of terminology here?

Reply:
changed it